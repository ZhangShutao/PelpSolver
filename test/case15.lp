a | b.
:- K[0, 1) a, K(0, 1] a.