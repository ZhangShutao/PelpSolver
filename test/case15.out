WorldView 1:
{[1.0]:{b}}
WorldView 2:
{[1.0]:{a}}